modules/subprojects.gradle

```diff
@@ -1,16 +1,33 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 // This magically allows subdirs in this subproject to themselves become sub-subprojects in a proper tree structure
 new File(rootDir, 'modules').eachDir { possibleSubprojectDir ->
-    def subprojectName = 'modules:' + possibleSubprojectDir.name
-    //println "Gradle is reviewing module $subprojectName for inclusion as a sub-project"
+    def subprojectName = ':modules:' + possibleSubprojectDir.name
     File buildFile = new File(possibleSubprojectDir, "build.gradle")
-    if (buildFile.exists()) {
-        println "Module $subprojectName has a build file so counting it complete and including it"
-        include subprojectName
-        def subprojectPath = ':' + subprojectName
-        def subproject = project(subprojectPath)
-        subproject.projectDir = possibleSubprojectDir
-    } else {
-        println "***** WARNING: Found a module without a build.gradle, corrupt dir? NOT including $subprojectName *****"
+    File moduleTxt = new File(possibleSubprojectDir, "module.txt")
+    if (!buildFile.exists()) {
+        logger.warn("***** WARNING: Found a module without a build.gradle, corrupt dir? NOT including $subprojectName *****")
```

Comment:
Trivial, but may want to either use lots of asterisks or none for both logging statements :-)

Reply:
`***** I DON'T KNOW HOW IMPORTANT THE ASTERISKS ARE MAYBE JENKINS IS COUNTING THEM OR SOMETHING`

Reply:
Haha nope, just me trying to spot my logging once upon a time when implementing it originally